embrace-config-schema.json

```diff
@@ -0,0 +1,274 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "JSON Schema definition for the Embrace configuration file.",
+  "id": "https://json.schemastore.org/embrace-config-schema-1.0.0.json",
+  "properties": {
+    "app_id": {
+      "description": "Your 5 character app ID.",
+      "type": "string",
+      "maxLength": 5,
+      "minLength": 5
+    },
+    "api_token": {
+      "description": "Your API 32-hexdigit token.",
+      "type": "string",
+      "maxLength": 32,
+      "minLength": 32
+    },
+    "ndk_enabled": {
+      "description": "Enables NDK crash capture. Defaults to false.",
+      "type": "boolean"
+    },
+    "sdk_config": {
+      "type": "object",
+      "properties": {
+        "automatic_data_capture": {
+          "type": "object",
+          "properties": {
+            "memory_info": {
```

Comment:
Is basically doesn't work and is deprecated (or will be). Should we not include this here so it's not suggested?

Reply:
can I remove it from the swazzler and SDK code?